egs/wsj/s5/steps/nnet3/tdnn/make_configs.py

```diff
@@ -535,6 +542,11 @@ def MakeConfigs(config_dir, splice_indexes_string,
                                                     self_repair_scale = self_repair_scale,
                                                     norm_target_rms = final_layer_normalize_target)
 
+            if final_mixture_window is not None and include_log_softmax:
```

Comment:
@freewym you should add final mixture even if there is no log_softmax layer. This is the main case we will be testing.